src/App.js

```diff
@@ -13,16 +14,25 @@ const toDoItems = [
 ];
 
 function App() {
+  const [toDoList, setToDoList] = useState([])
+
+  function addToDoList(title) {
+    let newItemArray = {id: "", title, completed: false};
+    let newToDoList = toDoList.slice();
+    newToDoList.push(newItemArray);
+    setToDoList(newToDoList);
+  }
+
   return (
     <div className="todo-app">
       <ToDoHeader header="My To Do App" />
 
       <ToDoWrapper>
-        <ToDoForm />
+        <ToDoForm onSubmit={ addToDoList } />
       </ToDoWrapper>
 
       <ToDoWrapper>
-        <ToDoList items={ toDoItems }/>
+        <ToDoList items={ toDoList }/>
       </ToDoWrapper>
     </div>
```

Comment:
Imo to całe się nadaje do wrzucenia do osobnego komponentu ToDo/ToDoList
App niespecjalnie powinien zawierać tak szczegółowe rzeczy